sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -276,6 +279,13 @@ def process(self, element):
     with self.assertRaisesRegex(ValueError, r'window.*None.*add_timestamps2'):
       pipeline.run()
 
+class AddTimestamp(beam.DoFn):
+  def process(self, element, timestamp=beam.DoFn.TimestampParam):
+    yield beam.window.TimestampedValue(element, timestamp)
+
+class GetTimestamp(beam.DoFn):
+  def process(self, element, timestamp=beam.DoFn.TimestampParam):
+    yield '{} - {}'.format(timestamp, element['name'])
```

Comment:
(Optional) I would either call this FormatWithTimestamp, or have it return a tuple.

Reply:
Done.